src/components/FlowRunTimeline.vue

```diff
@@ -287,33 +285,27 @@
       interval,
     )
 
-    const data = computed(() => {
-      return removeNullStartsAndSort(subscription.response ?? [])
-    })
+    const data = computed(() => subscription.response ?? [])
 
     expandedSubFlowRuns.value.set(id, {
       data,
       subscription,
     })
   }
 
-  function removeNullStartsAndSort(nodes: TimelineNode[]): GraphTimelineNode[] {
-    return nodes.filter(isValidGraphTimelineNode)
-      .sort((nodeA, nodeB) => {
-        return nodeA.start.getTime() - nodeB.start.getTime()
-      })
+  const getSubFlowRunIds = (data: GraphTimelineNode[]): string[] => {
+    return data
+      .map((node) => node.subFlowRunId)
+      .filter((subFlowRunId): subFlowRunId is string => subFlowRunId !== undefined)
   }
-
   const rootSubFlowRunIds = computed<string[]>(() => {
-    return graphData.value
```

Comment:
For some reason typescript wasn't convinced this had not been returning possibly `undefined`. Went ahead and changed the check so it typecasts `string[]`